editor/store/selectors.js

```diff
@@ -1744,3 +1744,21 @@ export function getSupportedBlocks( state, uid, globallyEnabledBlockTypes ) {
 export function getEditorSettings( state ) {
 	return state.settings;
 }
+
+/*
+ * Returns the editor settings.
```

Comment:
This comment seems inaccurate; maybe it was a copy/paste oversight?

This seems to just return the locking setting for this block.

Reply:
True I totally missed the update to the comment. Thank you for the catch.